.github/workflows/ci-obj-c.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 jobs:
   build_test:
     name: Build & Test Objective-C Wrapper
-    if: "! contains(github.event.head_commit.message, '[skip ci]')"
+    if: false
```

Comment:
Skipping because it runs on macos-11, which is not supported by github macos runners anymore. Attempted to upgrade it to macos-12 within a timebox, but wasn't successful: https://github.com/mattrglobal/ffi-bbs-signatures/pull/83

will schedule upgrading this action to macos-latest or 12 in a separate ticket, however for now, we have to skip this action to fix other issuers in the CI, which unblocks merging security vuln fixes.